### ISSUE TITLE
EmerDNS local and remote problems

### ISSUE THREAD

**Issue by NIR-Ginko** [emercoin/emercoin#77]
# Issue

I've built `emercoin-0.7.1`  on `FreeBSD-RELEASE-11.2` with `Clang` and I have a problem with DNS. I'm starting `emercoin-qt` with `emcdns=1` enabled but it does not open port 5335 so I can't query any domain name.

Here is my config file and `debug.log` file.
[emcconf.zip](https://github.com/emercoin/emercoin/files/2456994/emcconf.zip)

The application is running in IPv4-only network behind NAT.

I also tried to access `seed1.emercoin.com` and `seed2.emercoin.com` nameservers as specified on `OpenNIC` project page but with no success. Also `emergate.net` domain name is expired and I'm out of ideas how to configure this thing.

**Comment by maxihatop** [comment]
1. DNS server
Since you specified the config parameter 
emcdnsbindip=127.0.0.1

Your node connected to 127.0.0.1 only. See piece from your debug.log:
2018-10-07 22:29:00 EmcDns::EmcDns: Created/Attached: 127.0.0.1:5335; Qty=4:0

While, in the ordinary case, without this paramtert, it must be:
2018-10-05 02:29:28 EmcDns::EmcDns: Created/Attached: INADDR_ANY:5335; Qty=5:0

Just remove this line from the config, and server will accept request on any NIC.

2. emergate.net and seeds
We discontinue gatway emergate.net for 2 reasons:
a) We received many requests from copyright owners with claims about copyright infringement. on "our" site. Really, of course, those infringement was not in the emergate, but on the several sites, located in the network, available through our DNS. But we do not want to get problems from someone others activity, and decided to discontinue this gateway.
b) Several botnets used emergate and seeds for command and control purposes, with direct connections to our sites. To provide reasonable efforts for suppress this activity, we added activity limitation filters to DAP subsystem, and firewall, for answer to to OpenNIC servers only.

**Comment by NIR-Ginko** [comment]
1. @maxihatop , thank you for the response. Unfortunately, I tried to start `emercoin-qt` without `emcdnsbindip=127.0.0.1` and had no success too.

So, in the end I have to debug the issue by myself.

2. So, is it supposed that `OpenNIC` servers should resolve `.lib`, `.coin` and other zones successfully in case I try to query them? I tried querying `OpenNIC` servers using `dig` but I was unable to resolve any `.lib` domain names.
